src/lib/Libattr/master_sched_attr_def.xml

```diff
@@ -270,10 +270,10 @@
 	<member_name><both>ATTR_partition</both></member_name>		<!-- "partition" -->
 	<member_at_decode>decode_arst</member_at_decode>
 	<member_at_encode>encode_arst</member_at_encode>
-	<member_at_set>set_arst</member_at_set>
+	<member_at_set>set_arst_uniq</member_at_set>
```

Comment:
We are just making sure that no duplicate value is allowed for this attribute. Actual data validation of this attribute is happening in action_sched_port() function and hence I think it is fine to use set_arst_uniq() here.  I have also observed that similar mechanism is followed for ATTR_NODE_Mom("Mom") attribute of the node. Hope this is fine.

Reply:
You are correct, this is existing behavior for the mom attribute.  I personally disagree with its use, but since the code is already there, go ahead and use it.